### ISSUE TITLE
Bug: "Get-File" vs PHP 5.3 on Windows (low prio)

### ISSUE THREAD

**Issue by jrfnl** [shivammathur/setup-php#888]
### Describe the bug

When installing PHP 5.3, I'm seeing "Get-File: The term 'Get-File' is not recognized as a name of a cmdlet, function, script file, or executable program." errors in the logs (though they don't seem to be blocking anything)

### Version

- [X] v2
- [ ] v1

### Runners

GitHub Hosted

### Operating systems

`windows-latest`

### PHP versions

PHP 5.3

### To Reproduce

```yml
    runs-on: windows-latest

    steps:
      - name: Checkout code
        uses: actions/checkout@v4

      - name: Install PHP
        uses: shivammathur/setup-php@v2
        with:
          php-version: 5.3
          ini-values: 'error_reporting=-1, display_errors=On'
          extensions:
          coverage: none
        env:
          fail-fast: true
```

Note: `extensions` is an empty string as it is set dynamically from the matrix.

### Expected behavior

It all still seems to work fine, but I'm seeing the following errors in the logs:
```
==> Setup PHP 
Get-File: The term 'Get-File' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
Get-File: The term 'Get-File' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
√ PHP Installed PHP 5.3.29 
```

### Screenshots/Logs

![image](https://github.com/user-attachments/assets/2a47beac-d5b3-46f4-aec0-2518c6ac3e5e)

https://github.com/jrfnl/PHP_CodeSniffer/actions/runs/12100184185/job/33738527058

### Additional context

Would love to submit a PR, but am not familiar enough with Powershell unfortunately.

### Are you willing to submit a PR?

No

**Comment by shivammathur** [comment]
Fixed in https://github.com/shivammathur/setup-php/commit/d97b3754a8db2a1e6f7416f52eeb47d389de7690

**Comment by jrfnl** [comment]
Cricky.. that's a fast fix for a low prio item ;-) Thanks @shivammathur!

**Comment by shivammathur** [comment]
Released in [`2.32.0`](https://github.com/shivammathur/setup-php/releases/tag/2.32.0).
